lib/render.js

```diff
@@ -20,16 +20,19 @@ exports.toDisk = function(model) {
     process.exit(1)
   }
   if (opts.overwrite) {
+    var newModel = [];
     var list = require(process.cwd() + opts.overwrite).list;
     list.forEach(function(doc) {
       for(var index = 0; index < model.length; index++) {
-        console.log(model[index].name);
         if(model[index].name == doc.name && model[index].hierarchy[0] == doc.describe) {
           model[index].name = doc.newName;
-          model[index].summary = doc.newName;
+          model[index].hierarchy[1] = doc.newName;
+          model[index].summary = doc.newSummary;
+          newModel.push(model[index])
         }
       }
     });
+    model = newModel;
```

Comment:
Pick only test cases which are overwritten in `overwrite.json`. Before that docs command was documenting all the test cases.